woocommerce/woocommerce-gutenberg-products-block
# Mini Cart block: Add label for tax when it should be visible in accordance with WC settings

Mini Cart block: add a label for tax when it should be visible in accordance with WC settings




Fixes #4849

### Screenshots



![image](https://user-images.githubusercontent.com/4463174/142431758-57771966-e4c8-4181-bb85-90f28bacdb85.png)
*When `Display prices during cart and checkout` is set to `Including tax`*

![image](https://user-images.githubusercontent.com/4463174/142432020-cbfe895f-5c5b-4568-966c-19a3ab4be105.png)
*When `Display prices during cart and checkout` is set to `Excluding tax`*

### Testing

### Manual Testing

How to test the changes in this Pull Request:

Check out this branch.

1. Make sure you have Woo Blocks plugins installed.
2. Go to WooCommerce > Settings and check this option: 'Enable tax rates and calculations'.
3. In the Taxes tab, select 'No, I will enter prices exclusive of tax' and set 'Display prices during cart and checkout' to 'Including tax'.
4. Create a page with the `Mini Cart` block and save it.
5. Check that the `Mini Cart` block has no label when there isn't a product in the cart.
6. Check that the `Mini Cart` block, when there is at least one product in the cart, has `(incl. VAT)` or `(incl. tax)` label (it depends by the country).
